### DISCUSSION TITLE
blog_posts/performance_of_numerical_integrators

### DISCUSSION THREAD

**Discussion by giscus** [MissingDeadlines/iolite#79] [Blog Posts]
# blog_posts/performance_of_numerical_integrators

In this post we're going to take a look at different numerical integrators and how they behave with varying delta times.

https://iolite-engine.com/blog_posts/performance_of_numerical_integrators

**Comment by cipitaua** [comment]
"This fact makes it the most memory-hungry integrator up to this point."

actually this is not true if you split velocity verlet in four steps:
1) v = v + dt * a / 2
2) p = p + v * dt
3) update  a
4) v = v + dt * a / 2

no need to store anything
